seal_rookery/upload_images.py

```diff
@@ -118,14 +110,10 @@ def find_new_seals(access_key: str, secret_key: str) -> list:
     )
     s3 = session.resource("s3")
     bucket = s3.Bucket("seals.free.law")
-    aws_seals = [
-        x.key.split("/")[-1] for x in bucket.objects.filter(Prefix="v2/orig/")
-    ]
-    local_seals = [
-        x.split("/")[-1] for x in glob.glob(f"{ROOT_DIR}/seals/orig/*")
-    ]
+    aws_seals = [x.key.split("/")[-1] for x in bucket.objects.filter(Prefix="v2/orig/")]
+    local_seals = [x.split("/")[-1] for x in glob.glob(f"{ROOT_DIR}/seals/orig/*")]
```

Comment:
Reformatting from 79 char line width to Ruff/Black default of 88.

Happy to go back to 79 chars if you really want, but I think it's best to stick with the default.

Reply:
I have gone back on this and stuck to 79 to reduce churn in all repos when adopting Ruff. It doesn't make a big difference here but it will in larger repos, so let's stick with what you had.